src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -231,6 +233,15 @@ public StreamingResponseBody streamEvents(
         };
     }
 
+    /**
+     * Every consumer identifies itself using a client-id to use its quota. The client id is a combination of
+     * application name and event type so that every application can consume up to the quota limit per partition.
+     * Leader partitions from a single event type are guaranteed to be located on different brokers.
```

Comment:
@rcillo I guess it's fine not to document it in the code, but in that case it would be better to remove this line.